cmd/renterd/main.go

```diff
@@ -406,23 +416,30 @@ func main() {
 			return
 		}
 		setAPIPassword()
+	} else {
+		mustLoadAPIPassword()
```

Comment:
Aren't `setAPIPassword` and `mustLoadAPIPassword` the same thing? If so I'd get rid of one of them (probably the older one) and pull the call out of the `if` since it will be called in both cases.